themes/bootstrap3/templates/combined/results.phtml

```diff
@@ -73,7 +73,21 @@
       'showBulkOptions' => $this->showBulkOptions,
     ];
   ?>
-  <?=$this->context($this)->renderInContext('combined/stack-' . $placement . '.phtml', $viewParams); ?>
+
+  <?php if ($anySideRecommendations ?? false): ?>
+    <div class="<?=$this->layoutClass('mainbody')?>">
+      <?=$this->context($this)->renderInContext('combined/stack-' . $placement . '.phtml', $viewParams); ?>
+    </div>
+
+    <div class="<?=$this->layoutClass('sidebar')?>" id="search-sidebar">
+      <?php /* foreach ($this->results->getRecommendations('side') as $index => $current): ?>
```

Comment:
I don't think it hurts to offer this option, as long as we document it clearly. I'm even wondering if there might be some way of leveraging the Blender combined faceting in a recommendation module for combined search results, so you could have multi-column results with shared faceting. Seems possible in theory, though potentially impractical (and slow) in practice. In any case, that's a project for another day, but it would justify having this kind of setting available.

Reply:
Yeah I had the same thought about this.  I like the possibility (and simplicity) of enabling "simple" side facets that operate only on the combined search results, but the problem is it's useless for most of the recommendation modules.  Whereas if it was done in a smarter way like blended search, that would actually be a very nice addition.  But as you say that's a project for another day.

So I'm leaning to removing this option for now and considering it later in smarter way.

Reply:
That's fine with me. I'm always in favor of implementing things one piece at a time, so as long as we don't make it harder for ourselves to do this part later, there's no reason we have to do it right now.

Reply:
Yeah, I think adding this code now would actually make it harder for the feature to work 'well' in the future due to backward compatibility issues.  Will remove.